### PR TITLE
Staging/ad9361 Modify ad9361 project defines

### PR DESCRIPTION
**Pull request by danmois**
Remove duplicate code and rename DMA IRQ define.

### CODE REVIEW

File: projects/ad9361/src/main.c [hunk 1]
```diff
@@ -160,7 +160,7 @@ struct axi_dac_init tx_dac_init = {
 struct axi_dmac_init rx_dmac_init = {
 	"rx_dmac",
 	CF_AD9361_RX_DMA_BASEADDR,
-#ifdef ADC_DMA_IRQ_EXAMPLE
```
Comment (amiclaus) [inline projects/ad9361/src/main.c:163]: any specific reason for the rename? the previous versions looks more explicit for the user. Also, if you do this change please make sure that the .json file is updated accordingly.

**Review by amiclaus — COMMENTED** [review]
looks good, one comment.

File: projects/ad9361/src/main.c (hunk 1 above)
Comment (danmois) [reply projects/ad9361/src/main.c:163]: In the current implementation, the define is valid for both ADC and DAC, and not only for the first one. Therefore, removing ADC from the define makes the example selection clearer.

**Comment by danmois** [comment]
V2

- update builds.json with the new define for DMA IRQ

**Review by amiclaus — COMMENTED** [review]
you might want to squash the last 2 commits since they are interdependent.

File: projects/ad9361/src/main.c (hunk 1 above)
Comment (amiclaus) [reply projects/ad9361/src/main.c:163]: got it, thanks.

**Comment by danmois** [comment]
V3

- add modifications in builds.json to last commit

**Comment by danmois** [comment]
V4

- update DMA IRQ define in app_config.h

**Review by buha — APPROVED** [review]

**Review by amiclaus — APPROVED** [review]

**Pull request merged**